mgmt_cli_test.py

```diff
@@ -459,6 +470,55 @@ def get_email_data(self):
 
         return email_data
 
+    def test_backup_and_restore_only_data(self):
+        self.run_prepare_write_cmd()
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.get_cluster(cluster_name=self.CLUSTER_NAME) \
+            or manager_tool.add_cluster(name=self.CLUSTER_NAME, db_cluster=self.db_cluster,
+                                        auth_token=self.monitors.mgmt_auth_token)
+        backup_task = mgr_cluster.create_backup_task(location_list=self.locations)
+        backup_task_status = backup_task.wait_and_get_final_status(timeout=110000)
+        assert backup_task_status == TaskStatus.DONE, \
+            f"Backup task ended in {backup_task_status} instead of {TaskStatus.DONE}"
+        self.db_cluster.nodes[0].run_cqlsh('TRUNCATE keyspace1.standard1')
+        self.restore_data_with_task(mgr_cluster=mgr_cluster, backup_task=backup_task, timeout=110000)
+        self.run_verification_read_stress()
+
+    @staticmethod
+    def _get_persistent_snapshots():
+        with open("defaults/manager_persistent_snapshots.yaml", encoding="utf-8") as mgmt_snapshot_yaml:
+            persistent_manager_snapshots_dict = yaml.safe_load(mgmt_snapshot_yaml)
+        return persistent_manager_snapshots_dict
+
+    def test_restore_multiple_keyspaces(self):
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.get_cluster(cluster_name=self.CLUSTER_NAME) \
+            or manager_tool.add_cluster(name=self.CLUSTER_NAME, db_cluster=self.db_cluster,
+                                        auth_token=self.monitors.mgmt_auth_token)
+        if self.params.get('cluster_backend') != 'aws':
+            self.log.error("Test supports only AWS ATM")
+            return
```

Comment:
Why?
Even with the current implementation, couldn't you restore from AWS to GCP / Azure?

Reply:
Access to our bucket is limited by the IAM role, so I'm not sure we can even access the bucket from azure/gcp instances in the first place. Moreover, I'm sure the actual restore will take longer so we'd have to reconsider the timeouts that were already set, and for that I'll need to rerun the test to see the required times. So for now I chose to leave it at that.

Reply:
But don't we have tests for backup also in GCP and Azure?
If not, why would we even need to check the backend?

Reply:
We have a restore test in gce and azure. This specific test requires accessing an s3 bucket, which is only accessible from amazon instances. I added this check as a reminder